src/service/url-replacements-impl.js

```diff
@@ -956,12 +959,15 @@ export function extractClientIdFromGaCookie(gaCookie) {
 
 /**
  * @param {!./ampdoc-impl.AmpDoc} ampdoc
- * @return {!UrlReplacements}
  */
 export function installUrlReplacementsServiceForDoc(ampdoc) {
-  return getServiceForDoc(ampdoc, 'url-replace', doc => {
-    return new UrlReplacements(doc, new GlobalVariableSource(doc));
-  });
+  registerServiceBuilderForDoc(
+      ampdoc,
+      'url-replace',
+      /* opt_ctor */ undefined,
+      doc => {
+        return new UrlReplacements(doc, new GlobalVariableSource(doc));
+      });
```

Comment:
Nits: The param name is `opt_constructor` and you can use the short closure syntax here: `() => expr`.